BusinessLogic/LootLogic.cs

```diff
@@ -269,39 +269,266 @@ public async Task<bool> SetLootRequestStatus(int playerId, int currentPlayerId,
             return true;
         }
 
-        public LootOverviewViewModel GetOverview(int[] display, int syndicateId)
+        public LootOverviewViewModel GetOverview(int[] display, int tierSelect, int typeSelect, int syndicateId)
         {
             var lootStatuses = EnumExtensions.ToDictionary<int>(typeof(CampaignLoot.CampaignLootStatus));
             if (display == null || display.Length == 0)
             {
                 display = lootStatuses.Keys.ToArray();
             }
-            var model = new LootOverviewViewModel{
-                AllLoot = _dbContext.CampaignLoot
-                    .Include(l => l.Campaign)
-                    .Include(l => l.Holder)
-                    .Include(l => l.Item)
-                    .Join(
-                        _dbContext.Campaign,
-                        cl => cl.CampaignId,
-                        c => c.Id,
-                        (cl, c) => new { CampaignLoot = cl, Campaign = c }
-                    )
-                    .Where(clc => clc.Campaign.SyndicateId == syndicateId)
-                    .Select(clc => clc.CampaignLoot)
-                    .Where(cl => display == null || display.Contains((int)cl.Status)).OrderByDescending(cl => cl.Campaign.UTCDateTime),
-                OtherSyndicatesLoot = _dbContext.CampaignLoot
-                    .Include(cl => cl.Campaign)
-                    .ThenInclude(clc => clc.Syndicate)
-                    .Where(cl => cl.Campaign.SyndicateId != syndicateId && 
-                        cl.AvailableToOtherSyndicates == true &&
-                        (cl.Status == CampaignLoot.CampaignLootStatus.Undistributed ||
-                        cl.Status == CampaignLoot.CampaignLootStatus.StaysWithSyndicate))
-                    .ToList(), // ToList() is, sadly, necessary to persist syndicate info/tag.
-                LootStatuses = EnumExtensions.ToDictionary<int>(typeof(CampaignLoot.CampaignLootStatus)),
-                Display = display,
-            };
-            return model;
+            // And again, I can't enter conditional clauses into those queries translated directly into SQL
```

Comment:
You can, with ternary operator: `a ? b: c` and with adding more complex logical constructs into conditions.